api/spec/services/api_documentation/dry_types_to_json_spec.rb

```diff
@@ -178,7 +178,7 @@
           items: {
             type: "object",
             properties: {
-              name: { type: 'string' }
+              name: described_class.convert(types['string'])
```

Comment:
If the `Types::String` JSON output is changed in the future, this test will not require an update. It should have been written like this when I first wrote these tests.